packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -131,137 +74,177 @@ governing permissions and limitations under the License.
     *:focus {
       outline: none;
     }
+  
     .spectrum-Steplist-marker {
       border: none;
     }
   }
+  &.is-navigable > a {
+    cursor: pointer;
+  }
+  &.is-hovered {
+    &.is-navigable .spectrum-Steplist-marker {
+      border: none;
+    }
+  }
 }
-
-.spectrum-Steplist-label {
-  position: absolute;
-  left: 50%;
-  bottom: var(--spectrum-steplist-label-labelOffset);
-
-  display: block;
-  width: calc(var(--spectrum-steplist-step-width) * 1.5);
-
-  font-size: var(--spectrum-steplist-label-text-size);
-  white-space: normal;
-
-  transform: translateX(-50%);
+.spectrum-Steplist-item > a {
+  grid-area: link;
+  display: grid;
+  justify-items: center;
+  align-items: center;
+  grid-template-areas:
+    "number label";
+  
+  .spectrum-Steplist-marker {
+    grid-area: number;
+    outline: none;
+    box-sizing: border-box; /* So we can stroke nicely */
+    border-style: solid;
+    border-width: var(--spectrum-steplist-marker-border-width);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .spectrum-Steplist-label {
+    grid-area: label;
+  }
 }
 
-.spectrum-Steplist-markerContainer {
-  display: block;
-
-  z-index: 2;
-  position: absolute;
-  bottom: -10px;
-  left: 50%;
-
-  width: var(--spectrum-steplist-marker-hitArea);
-  height: var(--spectrum-steplist-marker-hitArea);
-
-  margin-left: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+.spectrum-Steplist--vertical {
+  .spectrum-Steplist-chevron {
+    display: none;
+  }
 }
 
-/* Circle step marker */
-.spectrum-Steplist-marker {
-  box-sizing: border-box; /* So we can stroke nicely */
-  display: block;
-
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  margin-top: calc(calc(var(--spectrum-steplist-marker-diameter) / 2) * -1);
-  margin-left: calc(calc(var(--spectrum-steplist-marker-diameter) / 2) * -1);
-
-  width: var(--spectrum-steplist-marker-diameter);
-  height: var(--spectrum-steplist-marker-diameter);
-
-  border-radius: var(--spectrum-steplist-marker-diameter);
+/* Generated item counts on all sizes except small */
+.spectrum-Steplist--medium,
+.spectrum-Steplist--large,
+.spectrum-Steplist--xlarge {
+  counter-reset: listItemCount;
+  li {
+    counter-increment: listItemCount;
+  }
+  .spectrum-Steplist-item {
+    &.is-complete .spectrum-Steplist-marker::before,
+    &.is-selected .spectrum-Steplist-marker::before {
+      content: counter(listItemCount);
+    }
 
-  border: 2px solid;
-  border-color: transparent;
-  background: transparent;
+    &.is-hovered {
+      &.is-navigable .spectrum-Steplist-marker::before {
+        content: counter(listItemCount);
```

Comment:
is this interational-able (making up a word there) it's not auto switching based off our storybook lang, but maybe we're not setting it in the right place. hard to tell, a 3min google didn't reveal anything either.

Reply:
Yeah, I have no clue how internationalizable this is.  It would clearly be a lot simpler to just put the item count into the DOM as a text node, but I didn't see that in the HTML examples of the W3C aria specs for this.  I think logically, the item count is provided by the `list-styles`'s `decimal` setting, but since you can't actually target those to style them, this is the only way to get the numbers there without inserting them into the DOM.  It would definitely make it simpler to do localization (or other things) if we were adding the node into the DOM with the count.  I'm totally happy to change it to the simpler, JS-based solution if that's cool with the RSP team.